charts/flyte-core/values-eks-override.yaml

```diff
@@ -0,0 +1,22 @@
+# -- # Flyte uses a cloud hosted Cron scheduler to run workflows on a schedule. The following module is optional. Without,
+# this module, you will not have scheduled launchplans/workflows.
+# Override eks value for using aws native scheduler
```

Comment:
We should add that we support native scheduler and this is for backward compatability to older scheduler only